docs/en/index.rst

```diff
@@ -73,6 +79,13 @@ annotations of a class. A common one is
 
 A reader has multiple methods to access the annotations of a class.
 
+Note that ``AnnotationRegistry::registerLoader('class_exists')`` only works
+if you already have an autoloader configured (i.e. composer autoloader).
+Otherwise, :ref:`please take a look to the other annotation autoload mechanisms <annotations>`.
+
+A reader has multiple methods to access the annotations
+of a class.
```

Comment:
This sentence is now duplicated above and below the note

Reply:
My bad... Thank you :) Fix in #320